docs/_posts/2019-05-19-seminar-minutes-2020-09.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Bot Friday 2020 年第 9 次活动纪要
+author: likai
```

Comment:
In order to show your author information in your blog post, you need also add yourself to <https://github.com/wechaty/bot5.club/blob/master/docs/_data/authors.yml>